packages/codemods/src/transforms/styled-components-to-compiled/styled-components-to-compiled.tsx

```diff
@@ -53,40 +54,55 @@ const transformer = (fileInfo: FileInfo, api: API, options: Options): string =>
     allowedImportSpecifierNames: imports.styledComponentsSupportedImportNames,
   });
 
-  const taggedTemplateExpressionsWithAttrs = collection.find(
-    j.TaggedTemplateExpression,
-    ({ tag }: TaggedTemplateExpression) => {
-      if (tag.type === 'CallExpression') {
-        const { callee } = tag;
-
-        return (
-          callee.type === 'MemberExpression' &&
-          callee.object.type === 'MemberExpression' &&
-          callee.object.object.type === 'Identifier' &&
-          callee.object.object.name === 'styled' &&
-          callee.property.type === 'Identifier' &&
-          callee.property.name === 'attrs'
-        );
-      }
-
-      return false;
-    }
-  );
-
-  if (taggedTemplateExpressionsWithAttrs.length) {
-    convertStyledAttrsToComponent({
-      j,
-      plugins,
-      expressions: taggedTemplateExpressionsWithAttrs,
-    });
-  }
-
   applyVisitor({
     plugins,
     originalProgram,
     currentProgram: collection.find(j.Program).get(),
   });
 
+  const styledImport = collection.find(
+    j.ImportSpecifier,
+    (specifier) => specifier.imported.name === 'styled'
+  );
+
+  styledImport.forEach((styledImport) => {
+    const isCompiledImport =
+      styledImport.parentPath.parentPath.value.source.value === COMPILED_IMPORT_PATH;
```

Comment:
could this cause a null pointer issue?

Reply:
I don't think so, the object structure is pretty standard. We can use something safer here or just wrap it in try/catch if you think that's necessary.

Reply:
Yeah maybe safer to do `styledImport?.parentPath?.parentPath`

Reply:
I think the main problem is `any` type - we've been bitten a couple times in the past because of this same thing coming out of `jscodeshift` :/